test/images/image-util.sh

```diff
@@ -77,7 +77,7 @@ build() {
     fi
 
     # copy the qemu-*-static binary to docker image to build the multi architecture image on x86 platform
-    if [[ $(grep "CROSS_BUILD_" Dockerfile) ]]; then
+    if grep -q "CROSS_BUILD_" Dockerfile; then
```

Comment:
```
In ./test/images/image-util.sh line 80:
    if [[ $(grep "CROSS_BUILD_" Dockerfile) ]]; then
          ^-- SC2143: Use grep -q instead of comparing output with [ -n .. ].
```